chichiwang/exercise-js
# Replace the logo location in the README

## Description
Linked to Issue: #81
PR #82 removed the logo from `static/images/logo.webp` since the application no longer uses it. However, the root `README.md` still references the logo file so it is being re-introduced in the root directory for the README file. The app will not use this file at all.

## Changes
* Introduce `logo.webp` file to the root directory of the project
* Update `README.md` to update the reference of the location of the logo image file

## Steps to QA
* Review this PR and use the GH menu to view the README file to ensure the logo appears
